pyatlan/generator/templates/methods/attribute/auth_policy.jinja2

```diff
@@ -0,0 +1,12 @@
+
+        @classmethod
+        # @validate_arguments()
+        def create(cls, name: str) -> {{ entity_def.name }}.Attributes:
+            if not name:
```

Comment:
Again, not clear to me how this is different than the lines just below (unclear why we'd need both validations / if really needed couldn't we just include this `not name` check in the `validate_required_fields` method?)

Reply:
I'm not sure how that ended up that way. I will add name to the validate_required_fields